Codeinwp/neve
# Hide the empty comment tag if comments are not enabled

### Summary
Used the comments_open() function and return early if its response is false.

### Will affect visual aspect of the product
NO


### Test instructions
- Try to disable one post's comment form
- Try to disable the comments sitewide
- Check if `<div id="comments" class="comments-area">` still appears in the code
-


Closes #2632.
